### ISSUE TITLE
ca_certs not recognized in gunicorn 19.3.0

### ISSUE THREAD

**Issue by technochakra** [benoitc/gunicorn#1122]
I am using gunicorn 19.3.0.   For testing purposes, I am trying to get gunicorn to listen on HTTPS using self signed certificates which I intend to replace with valid ones.  The command below gives an error and gunicorn does not start.

gunicorn --bind 0.0.0.0:8000 --workers=2 --certfile=/app/certs/webserver.pem --ca_certs=/app/certs/wwca.pem --log-level=debug  app_wsgi:app

Error:
usage: gunicorn [OPTIONS] [APP_MODULE]
gunicorn: error: unrecognized arguments: --ca_certs=/app/certs/wwca.pem

The pem file is available at the above location but I can't seem to figure out the reason for the error.  Also if remove ca_certs option, gunicorn starts in https mode.

I confined the connection is indeed https by connecting through openssl from another machine.
openssl s_client -connect w.x.y.z:8000
The certificate returned above is the one I am providing in the command line options.

So in summary, even though https works, I am unable to provide the CA file I created through the ca_certs option.  Is this a known issue or am I misinterpreting how to use the option?  Happy to provide more information if needed.

**Comment by berkerpeksag** [comment]
Thanks for the report. I think `--ca_certs` should be `--ca-certs`.

**Comment by technochakra** [comment]
You are correct.  --ca-certs works.  Thanks!

BTW the confusion may have been from the documentation @ http://gunicorn-docs.readthedocs.org/en/latest/settings.html#ca-certs 

I think I copied from the heading text and not the text in red font.  --help also says --ca-certs though throughout the documentation underscores replace the hyphen in the heading text.  

My issue is resolved so feel free to close the bug.  Not sure if the documentation confusion is really worth addressing or not.  

Thanks for the prompt response.

**Comment by berkerpeksag** [comment]
Yes, we could definitely make it more user friendly. Could you open a new issue for that?

Thanks!

**Comment by technochakra** [comment]
I'll open the new issue for documentation.
